acct-group/qbittorrent/qbittorrent-0.ebuild

```diff
@@ -0,0 +1,9 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit acct-group
+
+DESCRIPTION="Group for the system-wide net-p2p/qbittorrent server"
+ACCT_GROUP_ID=534
```

Comment:
how do you get this ID? I believe you should request it via gentoo api repo

please have a look at following repo, run "bin/used_free_uidgids.sh" to check available IDs, and send a PR
https://github.com/gentoo/api-gentoo-org

Reply:
https://github.com/gentoo/api-gentoo-org/pull/632